packages/app/screens/login.tsx

```diff
@@ -42,4 +42,5 @@ export const LoginScreen = withModalScreen(LoginModal, {
   matchingPathname: "/login",
   matchingQueryParam: "loginModal",
   snapPoints: ["90%"],
+  disableBackdropPress: true,
```

Comment:
Not sure about the login modal

Reply:
haha yeah I wasn't sure either 😂 . Should I remove it?